py/client/pydeephaven/session.py

```diff
@@ -33,52 +35,13 @@
 from pydeephaven.query import Query
 from pydeephaven.table import Table, InputTable
 
+_TRACE_ENABLED = False
+_MAX_REFRESH_CONSECUTIVE_FAILURES = 5
 
-class _DhClientAuthMiddlewareFactory(ClientMiddlewareFactory):
-    def __init__(self, session):
-        super().__init__()
-        self._session = session
-
-    def start_call(self, info):
-        return _DhClientAuthMiddleware(self._session)
-
-
-class _DhClientAuthMiddleware(ClientMiddleware):
-    def __init__(self, session):
-        super().__init__()
-        self._session = session
-
-    def call_completed(self, exception):
-        super().call_completed(exception)
-
-    def received_headers(self, headers):
-        super().received_headers(headers)
-        if headers:
-            auth_token = bytes(headers.get("authorization")[0], encoding='ascii')
-            if auth_token and auth_token != self._session._auth_token:
-                self._session._auth_token = auth_token
-
-    def sending_headers(self):
-        return {
-            **{
-                "authorization": self._session._auth_token
-            }, **self._session._extra_headers
-        }
-
-
-class _DhClientAuthHandler(ClientAuthHandler):
-    def __init__(self, session):
-        super().__init__()
-        self._session = session
-        self._token = b''
-
-    def authenticate(self, outgoing, incoming):
-        outgoing.write(self._session._auth_token)
-        self._token = incoming.read()
-
-    def get_token(self):
-        return self._token
-
+def trace(who):
```

Comment:
Consider removing this in favor of the logging import added above, which has a `debug` level.

Also consider creating a logger instance in this file, so that if the log format is set to log the name of the logger (rather like Java logging frameworks do), the log output will show that name.

That is, 
```
logger = logging.getLogger(__name__)
```

then replace `trace(...)` and `logging.warning(...)` with `logger.debug(...)` and `logger.warning(...)` respectively. The logger can be configured then to have a particular level, and a formatter (with timestamp, file name, frame that the call came from, etc).

Reply:
Done-ish.